### PR TITLE
Foreman 3.10 branching build updates

### PR DESCRIPTION
**Pull request by pcreech**

### CODE REVIEW

File: repoclosure/yum.conf [hunk 1]
```diff
@@ -109,38 +109,38 @@ failovermethod=priority
...
+[el7-foreman-client-3.10-staging]
 name=Foreman Client nightly EL7
-baseurl=https://stagingyum.theforeman.org/client/nightly/el7/x86_64/
+baseurl=https://stagingyum.theforeman.org/client/3.10/el7/x86_64/
 
-[el8-katello-nightly-staging]
+[el8-katello-3.10-staging]
 name=Katello nightly staging EL8
-baseurl=https://stagingyum.theforeman.org/katello/nightly/el8/x86_64/
+baseurl=https://stagingyum.theforeman.org/katello/3.10/el8/x86_64/
```
Comment (ekohl) [inline repoclosure/yum.conf:138]:
```suggestion
baseurl=https://stagingyum.theforeman.org/katello/4.12/el8/x86_64/
```

File: repoclosure/yum.conf [hunk 2]
```diff
@@ -109,38 +109,38 @@ failovermethod=priority
...
 module_hotfixes=1
 
-[el7-foreman-client-nightly-staging]
+[el7-foreman-client-3.10-staging]
 name=Foreman Client nightly EL7
-baseurl=https://stagingyum.theforeman.org/client/nightly/el7/x86_64/
+baseurl=https://stagingyum.theforeman.org/client/3.10/el7/x86_64/
 
-[el8-katello-nightly-staging]
+[el8-katello-3.10-staging]
```
Comment (ekohl) [inline repoclosure/yum.conf:136]:
```suggestion
[el8-katello-4.12-staging]
```

File: repoclosure/yum.conf [hunk 3]
```diff
@@ -109,38 +109,38 @@ failovermethod=priority
...
 name=Katello nightly staging EL8
-baseurl=https://stagingyum.theforeman.org/katello/nightly/el8/x86_64/
+baseurl=https://stagingyum.theforeman.org/katello/3.10/el8/x86_64/
 module_hotfixes=1
 
-[el9-katello-nightly-staging]
+[el9-katello-3.10-staging]
```
Comment (ekohl) [inline repoclosure/yum.conf:141]:
```suggestion
[el9-katello-4.12-staging]
```

File: repoclosure/yum.conf [hunk 4]
```diff
@@ -109,38 +109,38 @@ failovermethod=priority
...
 name=Katello nightly staging EL9
-baseurl=https://stagingyum.theforeman.org/katello/nightly/el9/x86_64/
+baseurl=https://stagingyum.theforeman.org/katello/3.10/el9/x86_64/
```
Comment (ekohl) [inline repoclosure/yum.conf:143]:
```suggestion
baseurl=https://stagingyum.theforeman.org/katello/4.12/el9/x86_64/
```

**Review by ekohl — CHANGES_REQUESTED** [review]
The repo names also contain nightly.

**Comment by pcreech** [comment]
@ekohl should be resolved now

**Review by ekohl — APPROVED** [review]

**Pull request merged**